cluster-provision/k8s/1.29/extra-pre-pull-images

```diff
@@ -14,6 +14,9 @@ quay.io/kubevirt/cdi-apiserver:v1.58.1
 quay.io/kubevirt/cdi-controller:v1.58.1
 quay.io/kubevirt/cdi-operator:v1.58.1
 quay.io/kubevirt/cdi-uploadproxy:v1.58.1
+docker.io/istio/install-cni:1.24.1
+docker.io/istio/pilot:1.24.1
+docker.io/istio/proxyv2:1.24.1
```

Comment:
I need to these images copied over to our quay account so that we don't hit any docker io limits

Reply:
The older images at the top of the file can be removed

```
quay.io/kubevirtci/install-cni:1.15.0
quay.io/kubevirtci/operator:1.15.0
quay.io/kubevirtci/pilot:1.15.0
```

Reply:
Asking to understand: @brianmcarey from what I know we have container image cache on CI, is it correct? 
Is so, do we still need mirroring if we have cache?